content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -23,6 +23,10 @@ following cases:
 [Spring Boot starter]:
   https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.build-systems.starters
 
+You can use the Spring Boot configuration files (`application.properties`,
+`application.yml`) to configure the OpenTelemetry Spring Boot starter. You can't
+do this with the OpenTelemetry Java agent.
+
```

Comment:
just checking, did you want to add this to the list above?

Reply:
for me, this is a weaker reason for using the OpenTelemetry starter than those mentioned above

Reply:
thanks, I think I'd suggest adding it to the end of the list, even if it's less important

* You can use the Spring Boot configuration files (`application.properties`,
`application.yml`) to configure the OpenTelemetry Spring Boot starter which doesn't work with the OpenTelemetry Java agent